halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/kubernetes/KubernetesEditAccountCommand.java

```diff
@@ -98,6 +98,50 @@ protected String getProviderName() {
   )
   private String removeOmitNamespace;
 
+  @Parameter(
+      names = "--kinds",
+      variableArity = true,
+      description = KubernetesCommandProperties.KINDS_DESCRIPTION
+  )
+  private List<String> kinds = new ArrayList<>();
+
+  @Parameter(
+      names = "--all-kinds",
+      description = "Set the list of kinds to cache and deploy to every kind available to your supplied credentials."
+  )
+  private boolean allKinds;
```

Comment:
I don't seem to see any references to allKinds, does this end up getting used anywhere? I tried enabling this as I wanted to allow spinnaker to allow all kinds of objects available in my k8 cluster, but it does not seem to pick up anything. E.g. I tried deploying a LimitRange and `Unsupported Kubernetes object kind 'LimitRange', unable to continue.`